content/projects/crystallize.md

```diff
@@ -0,0 +1,62 @@
+---
+title: Crystallize
+homepage: https://www.crystallize.com/
+twitter: CrystallizeAPI
+repo: https://github.com/CrystallizeAPI
+opensource: "No"
+typeofcms: "API Driven"
+supportedgenerators:
+    - All
+description: Crystallize is a GraphQL PIM service with a twist; manage structured product information and rich marketing content. A fast GraphQL API for building juicy headless customer experiences in any channel. Ideal for JAMStack & React. Sign up to your [free tier Crystallize account](https://crystallize.com/signup) and start building.
```

Comment:
```suggestion
description: Crystallize is a GraphQL PIM service with a twist; manage structured product information and rich marketing content. A fast GraphQL API for building juicy headless customer experiences in any channel.
```

Can you make the description a bit shorter so it fits better with the other cards around it? Added a suggestion to remove the link as the card doesn’t support markdown at the moment

Reply:
Screenshot for reference: 

<img width="1186" alt="Screen Shot 2020-07-06 at 2 17 15 PM" src="https://user-images.githubusercontent.com/6691035/86626239-693ea680-bf94-11ea-8ade-7956e210748d.png">

Reply:
Made the changes